src/main/resources/i18n/Resources_zh_Hant.properties

```diff
@@ -16,11 +16,9 @@ menuItemDefaultExit=關閉
 folderSelectorAlreadyUploadedLabelPrefix=已上傳:
 folderSelectorDirectoryChooserTitle=選擇媒體資料匣
 mainScreenAboutDialogTitle=關於
-#TODO translate (this one pops up when upgrading from an old version that does not support multiple upload methods)
-mainScreenInitialUploadMethodDialogTitle=You've got a choice!
+mainScreenInitialUploadMethodDialogTitle=正在使用舊版本上傳，舊版本不支持多重上傳方式
```

Comment:
This is just a short popup dialog title. What it is for is explained in the dialog itself. Could you please say "you have a choice" or "please make a choice" here?

Reply:
Same comment for the other file

Reply:
done